spyder_kernels/customize/spyderpdb.py

```diff
@@ -560,12 +604,43 @@ def do_debug(self, arg):
         argument (which is an arbitrary expression or statement to be
         executed in the current environment).
         """
+        with self.recursive_debugger() as debugger:
+            self.message("ENTERING RECURSIVE DEBUGGER")
```

Comment:
```suggestion
            self.message("Entering recursive debugger")
```

I like this message because it's informative but I don't think we have to display it all in caps. I mean, it can be a bit annoying if users enter into recursive debuggers a lot.